From 03e6a68cb9eb454411d927fede0fbf5189c78ceb Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Sun, 25 Jul 2021 15:46:54 +0200
Subject: [PATCH 092/N] Set MSYS2_ARG_CONV_EXCL for the shared Python module
 install

Otherwise one has to set it when calling "make install".
---
 Makefile.pre.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile.pre.in b/Makefile.pre.in
index 889a821..4e062e0 100644
--- a/Makefile.pre.in
+++ b/Makefile.pre.in
@@ -2331,6 +2331,7 @@ endif
 # Install the dynamically loadable modules
 # This goes into $(exec_prefix)
 sharedinstall: all
+	MSYS2_ARG_CONV_EXCL="--prefix=;--install-scripts=;--install-platlib=" \
 	$(RUNSHARED) $(PYTHON_FOR_BUILD) $(srcdir)/setup.py install \
 	   	--prefix=$(prefix) \
 		--install-scripts=$(BINDIR) \
